tests/console/yast2_samba.pm

```diff
@@ -183,11 +183,15 @@ sub setup_samba_startup {
         assert_screen 'yast2_samba-server_start-during-boot';
     }
     else {
-        change_service_configuration(
-            after_writing => {start         => 'alt-e'},
-            after_reboot  => {start_on_boot => 'alt-a'}
-        );
+        send_key 'alt-e';
+        send_key 'end';
+        send_key_until_needlematch 'yast2_ncurses_service_start_after_writing_conf', 'up', 5, 1;
```

Comment:
Have you created those needles or there were there before? I've suspected that it won't be trivial to reuse `change_service_configuration`, but if I wonder if we should still do that for this piece of code.

Reply:
Needles were there before. Again, That is the point of the ticket, it needs some investigation, as I could not figure out why that function was not working.

Reply:
Thing I don't get fully is if you managed to make it work manually and if it worth touching this module at all...